.env.example

```
@@ -11,6 +11,7 @@ NEXT_PUBLIC_SITE_URL=https://your-store.com
 NEXT_PUBLIC_STORE_NAME=Spree Store
 NEXT_PUBLIC_STORE_DESCRIPTION=A modern e-commerce storefront powered by Spree Commerce and Next.js.
 
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)